Loop.xcodeproj/project.pbxproj

```diff
@@ -1363,13 +1522,14 @@
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
 				CODE_SIGN_ENTITLEMENTS = Loop/Loop.entitlements;
 				CODE_SIGN_IDENTITY = "iPhone Developer";
-				DEVELOPMENT_TEAM = "";
+				DEVELOPMENT_TEAM = R6DR6B5C8B;
```

Comment:
fixed. sorry about that - hard to keep my branches straight when I'm deploying :-)